global/scripts/CorrThick.sh

```diff
@@ -0,0 +1,181 @@
+#!/bin/bash
+set -eu
+
+pipedirguessed=0
+if [[ ""$HCPPIPEDIR:-"" == "" ]]
+then
+	pipedirguessed=1
+	#fix this if the script is more than one level below HCPPIPEDIR
+	export HCPPIPEDIR="$(dirname -- "$0")/.."
+fi
+
+source "$HCPPIPEDIR/global/scripts/newopts.shlib" "$@"
+source "$HCPPIPEDIR/global/scripts/debug.shlib" "$@"
+
+#description of script/command
+opts_SetScriptDescription "Run curvature-corrected (folding-compensated) cortical thickness python script and save curvature-corrected (folding-compensated) thickness, curvatures, regression coefficients, and resampled outputs.
+
+References:
+  - https://www.biorxiv.org/content/10.1101/2025.05.03.651968v1
+  - https://onlinelibrary.wiley.com/doi/full/10.1002/hbm.25776
+Please cite these works when using this measure.
+
+Requirements:
+  Python dependencies: numpy, nibabel, scipy, psutil
+  Standard libraries: os, math, multiprocessing, concurrent.futures
```

Comment:
The requirements/dependencies are to tell the user what they need to install beyond a working python3.  A python install that is missing any standard libraries is not a usable python install in the first place.  We don't need to mention these.

Reply:
The dependencies that need to be installed by the user are fine to list.  The standard libraries used would already be installed, and don't tell the user anything they would need to know.  I'd like the help info to be be short and useful.